Source/WebCore/svg/SVGFECompositeElement.idl

```diff
@@ -35,6 +35,7 @@
     const unsigned short SVG_FECOMPOSITE_OPERATOR_ATOP       = 4;
     const unsigned short SVG_FECOMPOSITE_OPERATOR_XOR        = 5;
     const unsigned short SVG_FECOMPOSITE_OPERATOR_ARITHMETIC = 6;
+    const unsigned short SVG_FECOMPOSITE_OPERATOR_LIGHTER = 7;
```

Comment:
I am not sure what is more preferable here regarding indentation. But I think most of the SVG IDL use indentation for enum values. So let's have the new value also indented.